packages/orchestration/src/exos/stakingAccountKit.js

```diff
@@ -149,6 +240,49 @@ export const prepareStakingAccountKit = (baggage, makeRecorderKit, zcf) => {
             return this.facets.holder.delegate(validator, amount);
           }, 'Delegate');
         },
+        Deposit() {
+          trace('Deposit');
+          // TODO consider adding timeoutTimestamp as a parameter. current
+          // default is `FIVE_MINUTES_IN_SECONDS`
```

Comment:
```suggestion
          // default is 5 minutes
```

if you want to name the default, leave out the value
```suggestion
          // default is DEFAULT_TIMEOUT_SECONDS
```